src/common/ceph_context.cc

```diff
@@ -55,15 +55,22 @@ using ceph::HeartbeatMap;
 #include <pthread.h>
 
 #ifdef WITH_SEASTAR
+
+#include "seastar/core/shared_ptr.hh"
+#include "crimson/admin/context_admin.h"
```

Comment:
could you add an empty line after this `#include`?

Reply:
will do